ginkgo-reporters/polarion_reporter.go

```diff
@@ -33,53 +33,82 @@ import (
 var Polarion = PolarionReporter{}
 
 func init() {
-	flag.BoolVar(&Polarion.Run, "polarion", false, "Run Polarion reporter")
-	flag.StringVar(&Polarion.projectId, "polarion-project-id", "", "Set the Polarion project ID")
-	flag.StringVar(&Polarion.filename, "polarion-report-file", "polarion.xml", "Set the Polarion report file path")
+    flag.BoolVar(&Polarion.Run, "polarion-execution", false, "Run Polarion reporter")
```

Comment:
why did you replace tabs with spaces ?
https://blog.golang.org/go-fmt-your-code
I would stick to tabs since it default for go-fmt ...

Reply:
done :)